redCOMPONENT-COM/redSHOP
# REDSHOP-2157 Adding invoice_number chronological order

_Testing Info_
- Add `invoice_number` tag in invoice mail template.
- Place an order with any payment gateway, just remember that `invoice_number` will be only generated for confirm orders
- Check email and make sure you get `invoice_number`
- In backend order detail page, you will also able to find Invoice Number.
